ktor-utils/api/ktor-utils.api

```diff
@@ -491,8 +491,10 @@ public final class io/ktor/util/collections/ConcurrentList : java/util/List, kot
 	public fun clear ()V
 	public fun contains (Ljava/lang/Object;)Z
 	public fun containsAll (Ljava/util/Collection;)Z
+	public fun equals (Ljava/lang/Object;)Z
```

Comment:
Isn't it a public API change?

Reply:
Nope, the `equals` method is present by default. The difference is that we `override` it